pages/linux/zic.md

```diff
@@ -0,0 +1,12 @@
+# zic
+
+> Compile timezones into binary files.
+> More information: <https://manned.org/zic>.
+
+- Compile a timezone file from directory:
+
+`zic -d {{path/to/file}}.infile`
```

Comment:
```suggestion
- Compile a timezone file from a directory:

`zic -d {{path/to/directory}}`
```

Reply:
We have to select a file via the path for the command as per the description, so should it be changed to path/to/directory?